.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,23 @@
+## Details
+___Do not mention proprietary info or link to internal work items in this PR.___
+
+**What were the changes?**  
+_One sentence describing the work done._
+
+**Why were the changes made?**  
+_Explain the motivation behind the work. Provide any publicly-available historical context._
+
+**How was the outcome achieved?**  
+_Technical details behind the work. Explain any publicly-available hardware peculiarities._
+
+**Additional Documentation:**  
+_What else should the reviewer know?_
```

Comment:
I suggest this field is use to link JIRA or github issues

Reply:
No JIRA.

Reply:
Reason? JIRA is just a string like SWDEV-416920. What is the concern here?

Reply:
Your comment above says "link to JIRA".  No links to JIRAs should be provided.  Links to GitHub issues are ok.

Your string example is ok.  but be careful.  As people are in a rush to fill in details in PR, they may copy/paste the internal links, which is NOT good.  To avoid any mistakes, a "SAFE" way, is to avoid mentions to JIRA completely.  

Err on the side of caution, and avoid JIRA is best practice.

Reply:
I've addressed this elsewhere.